### PR TITLE
8328667: [Testbug] Enable ignored Spinner unit tests

### PR DESCRIPTION
**Pull request by karthikpandelu**
Enabled a test in `SpinnerTest` class.
Added comment to the editing test, why it can not be enabled. When Spinner value is set using `getEditor` `setText` method, the listener added to the spinner is not invoked. There is no bug for this issue. I'm not sure about the expectation in this case.
Other set of tests which are ignored are `LocalTimeSpinnerValueFactory` tests. Since these tests are unstable and fail only at certain time of the day, not enabling this test as well.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8328667](https://bugs.openjdk.org/browse/JDK-8328667): [Testbug] Enable ignored Spinner unit tests (**Bug** - P4)

### Reviewers
 * [Kevin Rushforth](https://openjdk.org/census#kcr) (@kevinrushforth - **Reviewer**)
 * [Andy Goryachev](https://openjdk.org/census#angorya) (@andy-goryachev-oracle - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jfx.git pull/1414/head:pull/1414` \
`$ git checkout pull/1414`

Update a local copy of the PR: \
`$ git checkout pull/1414` \
`$ git pull https://git.openjdk.org/jfx.git pull/1414/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1414`

View PR using the GUI difftool: \
`$ git pr show -t 1414`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jfx/pull/1414.diff">https://git.openjdk.org/jfx/pull/1414.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jfx/pull/1414#issuecomment-2011366829)

### CODE REVIEW

File: modules/javafx.controls/src/test/java/test/javafx/scene/control/SpinnerTest.java [hunk 1]
```diff
@@ -337,7 +338,7 @@ public void expectExceptionWhenOneArgsDecrementCalled_noValueFactory() {
      *                                                                         *
      **************************************************************************/
 
-    @Ignore("Need KeyboardEventFirer")
+    @Ignore("Need KeyboardEventFirer. Listener not invoked on changing value using setText method")
```
Comment (kevinrushforth) [inline modules/javafx.controls/src/test/java/test/javafx/scene/control/SpinnerTest.java:341]: Please file a bug for this and change the String to that bug ID, e.g., `@Ignore("JDK-nnnnnnn")`

Comment (karthikpandelu) [reply modules/javafx.controls/src/test/java/test/javafx/scene/control/SpinnerTest.java]: Created [JDK-8328701](https://bugs.openjdk.org/browse/JDK-8328701) and updated `@Ignore` comment

**Review by kevinrushforth — APPROVED** [review]

**Comment by kevinrushforth** [comment]
Looks good.

File: modules/javafx.controls/src/test/java/test/javafx/scene/control/SpinnerTest.java [hunk 2]
```diff
@@ -337,7 +338,7 @@ public void expectExceptionWhenOneArgsDecrementCalled_noValueFactory() {
      *                                                                         *
      **************************************************************************/
 
-    @Ignore("Need KeyboardEventFirer")
+    @Ignore("JDK-8328701")
```
Comment (andy-goryachev-oracle) [inline modules/javafx.controls/src/test/java/test/javafx/scene/control/SpinnerTest.java:341]: the spinner is not editable and no commitValue() is issued (please see my comment in the ticket)

https://bugs.openjdk.org/browse/JDK-8328701?focusedId=14659352&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14659352

Comment (kevinrushforth) [reply modules/javafx.controls/src/test/java/test/javafx/scene/control/SpinnerTest.java:341]: In that case, maybe you can repurpose that JBS bug ID into a test bug? Either way, using that bug ID to skip this test seems good.

**Pull request closed without merging**